src/rdkafka_conf.c

```diff
@@ -829,7 +829,10 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
 	  "Local message timeout. "
 	  "This value is only enforced locally and limits the time a "
 	  "produced message waits for successful delivery. "
-          "A time of 0 is infinite.",
+      "A time of 0 is infinite. "
```

Comment:
nit: indention seems off here (should be 8 whitespaces per tab), aligned with the lines above.